src/app.py

```
@@ -2,131 +2,141 @@
 openai_api_key = ""
 
 try:
     with open('C:\\Users\\alper\\Desktop\\mervesultan_openai_key.txt', 'r') as file:
         openai_api_key = file.read().strip()
-    logger.info("OpenAI API anahtarı başarıyla yüklendi.")
+    logger.info("OpenAI API key loaded successfully.")
 except Exception as e:
-    logger.error(f"OpenAI API anahtarı yüklenemedi: {e}")
-    raise RuntimeError(f"OpenAI API anahtarı yüklenemedi: {e}")
+    logger.error(f"Failed to load OpenAI API key: {e}")
+    raise RuntimeError(f"Failed to load OpenAI API key: {e}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)